mplc/dataset.py

```diff
@@ -17,9 +17,9 @@
 from librosa.feature import mfcc
 from loguru import logger
 from sklearn.model_selection import train_test_split
-from tensorflow.keras.datasets import cifar10, mnist, imdb
+from tensorflow.keras.datasets import cifar10, mnist, imdb, fashion_mnist
 from tensorflow.keras.layers import Activation
-from tensorflow.keras.layers import Conv2D, GlobalAveragePooling2D, MaxPooling2D
+from tensorflow.keras.layers import Conv2D, GlobalAveragePooling2D, MaxPooling2D, BatchNormalization
```

Comment:
Is batchNorm usable in federated context ? I asked myself the question back them but I did not really think about it, what do you think ?

Reply:
Is there a particular reason for it to not be usable in the context of FL? There's not much in the literature that addresses the impact of BatchNorm on the final aggregated model, at least from what i've seen.

Reply:
I wonder what is used for the BN weights of the global model, especially at the end for testing.
I am not sure either about what happen at the aggregation step
What do you think ?